### ISSUE TITLE
Debugging matching

### ISSUE THREAD

**Issue by mdomans** [foursquare/twofishes#95]
Here's my problem. I'm sending a JSON encoded requests to twofishes server. It's deployed from github repo as is - totally vanilla, no changes. Twofishes is used to map users locations as provided in their profiles to geographical points. That's mapped automatically, no human interaction.

One request I'm sending contains such query: **St. Paul** and the other request contains a rather similar query: **St. Paul, Minn.**.

The interesting part is that while the first one correctly maps to St. Paul in Minnesota, USA, the other maps to the city of Minden in Germany.

In other words, I'm asking is there any guide to debugging the differences between what I expect (a city in USA) and what I get (a city in Germany). 

At this point any document specifying what's correct and what's not would be rather useful.
